.gitignore

```diff
@@ -1,2 +1,3 @@
 /target/
 /Cargo.lock
+.DS_Store
```

Comment:
This should go into your system-local `.gitignore`.

Reply:
I was unaware of system-local `.gitignore`'s my mistake.
```suggestion
```